packages/iTwinUI-react/src/core/Wizard/Step.tsx

```diff
@@ -73,10 +79,11 @@ export const Step = (props: StepProps) => {
     }
   };
 
+  const initClassName = type === 'workflow' ? 'workflow-diagram' : 'stepper';
```

Comment:
I thought these will also be gone with separation.

Reply:
I meant to have separate components for steps for Stepper and WorkflowDiagram.

Reply:
There are now separate step components for Stepper and WorkflowDiagram